cubedash/_stac.py

```diff
@@ -1187,7 +1263,13 @@ def collection(collection: str):
     """
     Overview of a WFS Collection (a datacube product)
     """
-    return _stac_response(_stac_collection(collection))
+    # return _stac_response(_stac_collection(collection))
+    try:
+        _model.STORE.get_product(collection)
```

Comment:
Why does this line call `get_product`, and the return-statement calls `get_collection`?